src/v/serde/test/serde_test.cc

```diff
@@ -13,9 +13,21 @@
 #include "model/fundamental.h"
 #include "model/metadata.h"
 #include "random/generators.h"
+#include "serde/async.h"
+#include "serde/peek.h"
+#include "serde/rw/array.h"
+#include "serde/rw/bool_class.h"
+#include "serde/rw/bytes.h"
+#include "serde/rw/chrono.h"
+#include "serde/rw/enum.h"
+#include "serde/rw/envelope.h"
+#include "serde/rw/inet_address.h"
+#include "serde/rw/iobuf.h"
+#include "serde/rw/map.h"
 #include "serde/rw/rw.h"
+#include "serde/rw/set.h"
+#include "serde/rw/sstring.h"
```

Comment:
The header `serde/serde.h` is not exposed as a bazel target. I prefer the more granular approach where we have to define the headers manually, so I preferred this solution over defining a target for that header.

I could be convinced otherwise, though.

Reply:
> The header serde/serde.h is not exposed as a bazel target.

This is intention. There are not many uses of `serde.h` left in the tree. As soon as they are all gone, let's delete `serde.h`.

Reply:
Sounds good to me!